test/fixtures/output/schema-registry/config/delete-config.golden

```diff
@@ -0,0 +1,3 @@
++---------------------+----------+
+| Compatibility Level | BACKWARD |
```

Comment:
Shouldn't we print a message along the lines of "Schema Registry configuration successfully deleted" to match the style of our other commands?